GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2968,7 +2968,7 @@ private void LaunchRebase(string command)
 
             string rebaseCmd = Commands.Rebase(
                 GetActualRevision(LatestSelectedRevision)?.FirstParentId?.ToString(), interactive: true, preserveMerges: false,
-                autosquash: false, autoStash: true, ignoreDate: false, committerDateIsAuthorDate: false, supportRebaseMerges: Module.GitVersion.SupportRebaseMerges);
+                autosquash: false, autoStash: true, ignoreDate: false, committerDateIsAuthorDate: false, supportRebaseMerges: Module.GitVersion.SupportRebaseMerges, updateRefs: false);
```

Comment:
Make it the default argument value?

Reply:
Can we please create a struct that contains all the parameters instead? Not only it'd be more maintainable; there's a threshold on a number of parameters (I don't know it by heart) after which every new parameter becomes an overhead.

Reply:
@RussKie Not sure that I agree with this point of view (even if I understand that there are a lot of params due to the number of options of the git rebase command) because:
* it's consistent with all the methods of the `Commands` class. I would say it's even the goal of this class to use this pattern where you have a param for each command option.
* create a struct/record will have the same problem where you initialize a lot of values. You just add a level of indirection
* it's acceptable because the command is used only 3 times and I don't it will increase in the future and that's not a call that we modify often. I think we can live with that.

@RussKie I don't really get what you means by overhead.
Is it a performance one? In this case it is not a problem because it is called once and waiting for human interaction.
Or cognitive one? If that's the case, we say that the limit is around 7 +/-2. But as it is a simple mapping, with a very simple logic and without interaction between the parameters, the code logic is very easy to understand and a much bigger number is acceptable.